src/kvm/tracee.rs

```diff
@@ -21,6 +21,7 @@ pub struct kvm_msrs {
     pub nmsrs: u32,
     pub pad: u32,
     //
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
     pub entries: [kvmb::kvm_msr_entry; 1],
 }
```

Comment:
I think the whole struct can be made x86 only. KVM_GET_MSRS does not exists on other platforms.

Reply:
Ok.